list_of_standing_committees_and_working_groups.md

```diff
@@ -29,3 +29,7 @@ Charter Summary: License, protect, and promote the trademarks and visual and tex
 ### Jupyter Community Building
 
 [Charter](communitybuildingworkinggroup.md) Summary: Grow, build, and connect the global Jupyter community of users and contributors.
+
+### Jupyter Media Strategy
+
+[Charter][charters/MediaStrategyCharter.md] Summary: Ensure that communications in Jupyter official channels are strategic and benefit Project Jupyter.
```

Comment:
Misc on the gist you post. 

1) I would be worry of a google group, it's one more channel to manage WRT credential and access, can this be a GH team and github discussion?
2) missing Twitter/`@IPythonDev` ? And we used to have IRC... should we try to re-get access, and jupytercon.slack.com as well.

Reply:
- We prefer the use of a Google Group for communication because it allows for us to have non-public conversations if needed before making it public, and it follows the pattern used by other working groups and subprojects.
- We added Twitter/@IPythonDev, thanks!
- We have no intention of handling IRC on this group, and aren't aware of any existing accounts or who would have credentials.
- We have deferred considering Slack as a long term solution due to the cost.  We may consider Discord or Zulip.